platform/broadcom/sai.mk

```diff
@@ -1,7 +1,7 @@
 LIBSAIBCM_XGS_VERSION = 13.2.0.4
-LIBSAIBCM_DNX_VERSION = 12.3.3.2
+LIBSAIBCM_DNX_VERSION = 12.3.20.2
 LIBSAIBCM_XGS_BRANCH_NAME = SAI_13.2.0_GA
-LIBSAIBCM_DNX_BRANCH_NAME = SAI_12.3.0_master
+LIBSAIBCM_DNX_BRANCH_NAME = SAI_12.3.0_GA 
```

Comment:
@vmittal-msft - Was the dnx image validated with this change?  Since the SAI header has moved forward, please verify the compatibility with 12.3